### PR TITLE
Add extra support for GitHub, Trello and Jenkins

### PR DESCRIPTION
**Pull request by pedro-teixeira**
What's inside:

- Upgrade version to 0.2.0
- Upgrade dependencies 
- Improve documentation
- GitHub:
    - Validate requests
    - Assign user
    - Set commit status
- Trello:
    - Validate requests
    - User assigned hook
    - Attach URL to card
- Jenkins:
    - Commit build hook

### CODE REVIEW

**Comment by sgabler** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -50,11 +56,14 @@ var events = porcupine.getEvents();
...
+    oauthToken : 'your-token',
+    secret : 'your-secret'
   },
   trello : {
-    key : 'your_key',
-    token : 'your_token'
+    key : 'your-key',
+    token : 'your-token',
+    secret : 'your-secret',
+    callBackUrl 'https://your-porcupine-url/trello'
```
Comment (azylinski) [inline README.md]: No ` : ` here?

**Comment by azylinski** [comment]
:+1:

**Comment by pedro-teixeira** [comment]
Fixed.

**Comment by sergejjevsejev-home24** [comment]
time for new release!!! :smile:

**Comment by sjevs** [comment]
it is time for new release! :smile:

**Comment by pedro-teixeira** [comment]
Sorry for pushing the version up, Stefan could kill me for that.

**Comment by stefanjudis** [comment]
:bowtie:

**Pull request merged**